experimental/linting_tool/lib/pages/home_page.dart

```diff
@@ -3,13 +3,74 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:linting_tool/model/rules_store.dart';
+import 'package:linting_tool/layout/adaptive.dart';
+import 'package:linting_tool/widgets/lint_expansion_tile.dart';
+import 'package:provider/provider.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
   const HomePage({Key? key}) : super(key: key);
 
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
```

Comment:
I'm curious what state `_HomePageState` is holding on to. I don't see any fields here, nor a `setState`. What am I missing?